README-minishift.md

```diff
@@ -188,17 +188,31 @@ oc create clusterrolebinding federation-admin \
     --serviceaccount="federation-system:default"
 ~~~
 
-Deploy the [cluster registry](https://github.com/kubernetes/cluster-registry) and the namespace where clusters are registered
-(`kube-multicluster-public`):
+Clone the Federation V2 repo and checkout the tag 0.0.4:
 
 ~~~sh
-oc create -f cluster-registry.yaml
+cd federation-v2/
+git checkout tags/v0.0.4
```

Comment:
Even if the checkout is not needed (if the commit is added as per the previous comment), the `cd` should stay